apps/download-service/src/app/modules/regulation-documents/regulation-documents.controller.ts

```diff
@@ -21,8 +21,10 @@ import {
   ScopesGuard,
 } from '@island.is/auth-nest-tools'
 import { GetRegulationDraftDocumentDto } from './dto/getRegulationDraftDocument.dto'
-import { RegulationPdfInput } from '@island.is/regulations/web'
-import { RegName, toISODate } from '@island.is/regulations'
+import {
+  RegulationPdfInput,
+  DB_RegulationDraft,
```

Comment:
This type does not exist anymore.
Something something RegulationDraftModel comes in its place.
See @Valur's recent work.

Reply:
Nevermind, this has been fixed.